src/commcare_cloud/ansible/roles/upgrade_warning/tasks/main.yml

```
@@ -36,5 +37,12 @@
        - https://forum.dimagi.com/t/commcare-cloud-release-notes-2023-03-29/9739
        - https://commcare-cloud.readthedocs.io/en/latest/changelog/0071-upgrade_to_ubuntu_22.04.html
 
+      If you really know what you're doing, you can silence this
+      warning by adding
+
+      --skip-tags "upgrade_warning"
```

nit: quotes not needed
```suggestion
      --skip-tags upgrade_warning
```

---

They seem to be always used in official examples (e.g. https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_tags.html). But, yes, it should work either way.